### DISCUSSION TITLE
Is it possible to save a script on the camera?

### DISCUSSION THREAD

**Discussion by dimitribeyondgravity** [gopro/labs#475] [Q&A] [answered]
Hi,

It would be useful to be able to keep a script on the camera even if the SD Card is formatted.
But as the script is in the MISC folder, i guess it's not possible, and the only way is too scan a qr code again, containing this script.

Or somebody sees another solution?

Thanks !

**Comment by dnewman-gpsw** [comment, marked as answer]
Saving the script to the SD card is for user safely. I bad script that runs at boot, a script that crashes, has no fall-back.  The same bad script running from an SD card, eject the SD and boot, the camera is fine.  It is necessary that the script be in an eject-able form.
